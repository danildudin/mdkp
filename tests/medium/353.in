30 10 0
924 632 604 511 855 869 423 958 695 437 149 761 172 579 979 707 463 494 521 406 215 396 808 332 895 560 953 896 471 169 
875 855 94 636 297 475 676 231 244 560 533 871 909 868 394 79 237 171 445 888 70 987 117 577 436 978 725 803 793 185 
709 235 465 555 669 578 877 401 862 926 466 305 69 928 37 152 578 735 286 466 893 50 865 410 820 313 574 177 92 876 
521 722 537 191 681 858 13 779 806 354 736 201 73 459 78 605 226 705 18 676 294 411 353 292 379 505 358 785 636 184 
583 188 836 18 815 880 815 246 53 117 722 588 195 528 275 231 229 707 587 29 83 558 496 489 176 484 602 255 409 866 
944 23 164 497 869 363 788 457 743 848 832 705 208 403 410 132 678 745 912 672 227 645 904 754 422 583 652 215 486 698 
413 511 321 338 15 940 305 33 241 17 489 548 186 882 111 993 401 161 223 25 717 515 792 295 860 222 345 369 960 450 
357 545 926 660 876 45 256 775 356 777 122 291 276 545 780 475 660 598 196 900 208 788 68 289 950 551 215 359 873 770 
632 78 511 979 775 71 444 524 266 313 803 27 191 777 7 112 270 3 493 145 142 515 435 624 555 372 529 848 333 88 
334 248 995 743 130 260 541 623 552 157 169 326 364 129 704 821 596 631 892 358 386 189 495 966 738 855 155 397 477 692 
505 871 871 863 982 736 87 834 875 442 959 571 127 916 145 312 129 743 334 271 671 626 20 50 576 68 917 187 792 251 
7241 9299 6925 8918 7300 7818 9446 6510 6351 5324 
