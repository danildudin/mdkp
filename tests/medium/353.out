9842
13
0 4 5 11 12 14 15 16 22 24 25 26 27 
